indico/modules/events/registration/schemas.py

```diff
@@ -126,7 +132,12 @@ def _get_registration_data(self, registration):
             # File field stores the uuid as data which is not helpful.
             # We want to show the filename instead.
             if field['id'] in filenames:
+                obj = registration.data_by_field[field['id']]
                 field_data['data'] = filenames[field['id']]
+                if field['inputType'] == 'picture':
+                    field_data['data'] = url_for('.registration_picture', obj.locator.file, _external=True)
```

Comment:
What do you think about using `picture_url` as the key for this? Feels a bit cleaner than reusing `data` for it

Reply:
This will cause a lot of changes in the PWA (not that there won't be already with this PR) - see my latest commit